themes/src/main/resources/theme/keycloak-preview/account/resources/README.md

```diff
@@ -0,0 +1,48 @@
+New Account Console
+========
+
+The new Account Console uses PatternFly 4 and React.
+
+Here is what you need to know in order to do development on the new console.
+
+
+Building
+--------
+First, you must install node and npm.
+
+To build and lint the application:
+
+`npm run install`
+
+If your IDE doesn't handle dynamic TypeScript transpilation, use the watch script:
+
+`npm run watch`
```

Comment:
`npm run watch` gives me nothing. But if I try instead `npm run build:watch` it works for me. I believe is just the matter of tweak `package.json` to give what you would like into this readme.

Reply:
Right again.